src/_scss/components/typography/_global.scss

```diff
@@ -103,3 +103,18 @@ p + h2, p + h3, p + h4, p + h5, p + h6 {
    margin-top: $base-line-height * 3;
 }
 
+
+
+// links (underline if surrounded by other text)
+ul,
+ol,
+p {
```

Comment:
Normally we do these commas on the same line, not different ones. Let's stay consistent with this

Reply:
I was following the top of the file
```
// headers
h1,
h2,
h3,
h4,
h5,
h6,
.display-1,
.display-2 {
    font-family: $font-sans;
    color: #555;
    margin-bottom: $base-line-height;
}
```

Want me to update that too?

Reply:
Yeah, I looked at a couple other files before commenting that and I think most of the time it's the single line. I do wonder if they did that because it got too long in this case but if it fits on the one line then I think yeah, update that one too